spec/features/proposal/search/collection_search_spec.rb

```diff
@@ -62,6 +62,23 @@
           expect(page).to have_link('DIF 9', href: download_collection_xml_path(@ingest_response['concept-id'], 'dif'))
           expect(page).to have_link('DIF 10', href: download_collection_xml_path(@ingest_response['concept-id'], 'dif10'))
         end
+
+        context 'when trying to download data' do
```

Comment:
interesting that it worked this way, but I like that we are able to actually test it somehow.